telegram/ext/_application.py

```diff
@@ -1431,12 +1432,15 @@ def add_handlers(
         .. versionadded:: 20.0
 
         Args:
-            handlers (List[:class:`telegram.ext.BaseHandler`] | \
-                Dict[int, List[:class:`telegram.ext.BaseHandler`]]): \
+            handlers (Sequence[:class:`telegram.ext.BaseHandler`] | \
+                Dict[int, List[:class:`telegram.ext.BaseHandler`] | \
+                Tuple[:class:`telegram.ext.BaseHandler`]]):
```

Comment:
```suggestion
                Dict[int, Sequence[:class:`telegram.ext.BaseHandler`]]):
```
should this be updated as well?
Do you get any type checker problems when passing a dictionary?

Reply:
No I did not get any type checker problems.

I just updated this documentation part to match the actual function type hint.

Reply:
mh, using 

```python
app.add_handlers({1: mixed_handlers})

app.add_handlers({1: command_handlers})
```

in your MWE in #4530, I get 

```shell
mypy foo.py
foo.py:50: error: Dict entry 0 has incompatible type "int": "List[CommandHandler[CallbackContext[ExtBot[None], Dict[Any, Any], Dict[Any, Any], Dict[Any, Any]], None]]"; expected "int": "Union[List[BaseHandler[Any, CallbackContext[ExtBot[None], Dict[Any, Any], Dict[Any, Any], Dict[Any, Any]], Any]], Tuple[BaseHandler[Any, CallbackContext[ExtBot[None], Dict[Any, Any], Dict[Any, Any], Dict[Any, Any]], Any]]]"  [dict-item]
Found 1 error in 1 file (checked 1 source file)
```

changing the type hint for the dict values to sequence fixes the problem. So I'd suggest to update that as well

Reply:
Oh...

When I tested I added to the MWE this:

```python
app.add_handlers(
    {
        1: [
            CommandHandler("hello", hello),
            MessageHandler(filters.TEXT & ~filters.COMMAND, echo),
        ]
    }
)

app.add_handlers(
    {
        1: [
            CommandHandler("hi", hi),
            CommandHandler("welcome", welcome),
        ]
    }
)

```
This, for some reason, does not trigger any mypy errors.

Reply:
But thanks for the catch too!

Will send a update right away!

Reply:
Done!